### DISCUSSION TITLE
icon can't set in my windows app

### DISCUSSION THREAD

**Discussion by ssz25** [flet-dev/flet#2937] [Q&A]
flet pack is OK  ,  set the app icon ,  on windows app

but  I use cx_Freeze  to pack my app  ,  the app icon is can't set 

window title icon  
window task bar icon

Hope to add icon setting properties as soon as possible

**Comment by ndonkoHenri** [comment]
This is actually not a Flet issue. Search on how that is done in cxFreeze...
